.tests/facebook.sh

```diff
@@ -1,22 +1,22 @@
 #!/bin/bash
 
 # Check Facebook handle
-check_facebook () {
-  handles="$(git --no-pager diff origin/master..HEAD ../_data | grep ^+[[:space:]] | grep facebook | cut -c16-)"
+check_facebook() {
+  handles="$(git log -p origin/master..HEAD ../_data | grep "^+[[:space:]]*facebook: " | cut -c16-)"
 
-  if [ -z "$handles" ]; then
+  [[ -z "$handles" ]] && {
     echo "No Facebook handles found."
     exit 0
-  fi
+  }
```

Comment:
Why, oh why? That if/then/fi was fine, now it's an eye-sore.

Reply:
This works the same way. Just less characters.